### ISSUE TITLE
[BUG]  duplicate upstream error in 1.5.0-beta 

### ISSUE THREAD

**Issue by flexstat** [bunkerity/bunkerweb#466]
**Description**
I try this example https://github.com/bunkerity/bunkerweb/blob/v1.5.0-beta/examples/load-balancer/docker-compose.yml

At the first start, everything is ok, an upstream.conf.conf file is created after I stop the container and start it again, SCHEDULER creates a new upstream.conf.conf and then the system returns duplicate upstream error 

**Logs**
```
"app" in /etc/bunkerweb/configs/http/upstream.conf.conf:2
mybunker_1 | nginx: [emerge] duplicate upstream "app" in /etc/bunkerweb/configs/http/upstream.conf.conf:2
```

**Comment by fl0ppy-d1sk** [comment]
Hello @flexstat,

Thanks for your report.

We have a fix for this issue, it will be soon integrated into the beta.

BTW, if you remove the `.conf` suffix in the env var `CUSTOM_CONF_HTTP_upstream` it should work.

**Comment by fl0ppy-d1sk** [comment]
Hello @flexstat,

This is fixed in the last [1.5.0-beta realease](https://github.com/bunkerity/bunkerweb/tree/v1.5.0-beta).

Thanks again !
